core/extensions.js

```diff
@@ -176,6 +176,41 @@ Blockly.Extensions.checkDropdownOptionsInTable_ =
   };
 
 /**
+ * Builds an extension function that will install a dynamic tooltip. The
+ * tooltip message should include a the string '%1' and that string will be
```

Comment:
Hi, I have one query. Blockly.Msg.MATH_CHANGE_TOOLTIP in math.js and Blockly.Msg.CONTROLS_FOR_TOOLTIP, Blockly.Msg.CONTROLS_FOREACH_TOOLTIP in loops.js are having undefined values.